packages/theme/modules/catalog/product/composables/useGallery.ts

```diff
@@ -0,0 +1,21 @@
+import { computed } from '@nuxtjs/composition-api';
+import { getGallery as getProductGallery } from '~/modules/catalog/product/getters/productGetters';
+import { useImage } from '~/composables';
+import { Product } from '~/modules/catalog/product/types';
+
+export function useGallery(product: Product) {
```

Comment:
```suggestion
export function useGallery(product: Ref<Product>) {
```

Is this reactive? I remember when I was creating useProductsWithCommonCardProps.ts I needed to pass a ref through the argument, not a regular value

Reply:
I don't understand ;) this is MY NEW composable ;) Param is not a ref but the output is a computed value (at least partially).